# The SICK Laser discription file of the robot in habitat
hokuyo:

  type: "Laser"
  name: "hokuyo"
  frame: "/laser_hokuyo"
  topic: "/scan3"
  publish_transfromation: True
  position: [-0.15, 0.0, 0.1]
  orientation: [0.0, 3.14159, 0.0]

  sensor_info:

    far: 5.6
    near: 0.06

    unit: "deg" # def or rad
    ang_min: -100
    ang_max: 100
    ang_increment: 0.36

    mean_error: 0.03
    max_error: 0.04
    rate: 10

    # not used for now
    hfov: 240
    resolution:
      horizontal: 1080
      vertical: 1080 # must be the same to horizontal resolution to force habitat rendering the horizontal as the expected resolution.